java/arcs/android/devtools/StoreMessage.kt

```diff
@@ -172,6 +60,9 @@ class StoreMessage(
                     else -> JsonValue.JsonString(value.valueRepr)
                 }
             }
+            else -> {
+                return JsonValue.JsonString(value.toString())
+            }
```

Comment:
If `null` is passed to this method, I think this `else` will mean that the returned value will be `JsonString("null")` rather than what I assume is intended by the last line of the method: `JsonNull`

Reply:
Changed so the (now) extension function takes a Referencable. We handle cases where null could be separately. (It's only 1 place)